libraries/engage/product/components/MapPriceHint/connector.js

```diff
@@ -1,15 +1,17 @@
 import { connect } from 'react-redux';
-import { getProductPriceData } from '@shopgate/pwa-common-commerce/product/selectors/product';
+import { makeGetProductPriceData, makeGetProductMapPrice } from '../..';
```

Comment:
Importing from the root index file from within the same library is not a good idea. This creates a circular dependency and slows down unit tests as well.